js/app.js

```diff
@@ -1,52 +1,71 @@
-let output = document.getElementById("outputContainer")
-
+// Input : the first display line on the screen
 let input = document.getElementById("input")
 
+// Input : the second display line on the screen
+let output = document.getElementById("outputContainer")
+
 function display(num){
 
     if(input.value.length > 6){
-        document.getElementById("input").style.fontSize = "48px";
+        input.style.fontSize = "48px";
     }
     
     if(input.value.length > 12){
-        document.getElementById("input").style.fontSize = "28px";
+        input.style.fontSize = "28px";
     }
     
     if(input.value.length > 20){
         input.value = "";
     }
 
     input.value += num;
+ 
     console.log(input.value.length); 
     
 }
 
-function clear(){
+// Clear function called in AC button which clears the screen
+function Clear(){
```

Comment:
function name should not start with capital letter. We follow camelCase for function name.